client/js/components/statement/splitStatement/SplitStatementView.vue

```diff
@@ -485,12 +483,47 @@ export default {
         this.prosemirror.view,
         rangeTrackerKey,
         editStateTrackerKey,
-        id,
-        { active: this.editingSegment.charEnd, fixed: this.editingSegment.charStart }
+        id
       )
       this.ignoreProsemirrorUpdates = false
     },
 
+    extractHtmlWithRangeMarks () {
```

Comment:
shouldn't this method be obsolet when prosemirror knows about the segmentMark-type? I thought the toDom/fromDom methods there exactly handle this.